app/src/pages/playground/PlaygroundRunButton.tsx

```diff
@@ -25,12 +52,16 @@ export function PlaygroundRunButton() {
       }
       size="S"
       onPress={() => {
-        if (isRunning) {
-          cancelPlaygroundInstances();
-        } else {
-          runPlaygroundInstances();
-        }
+        toggleRunning();
       }}
+      trailingVisual={
+        <Keyboard>
+          <VisuallyHidden>command</VisuallyHidden>
```

Comment:
What about windows/Linux? Ctrl+Enter should be used there.

Reply:
added useModifierKey to detect if Mac